startups/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from rest_framework.routers import DefaultRouter
+from startups.views import StartupSearchViewSet
+
+router = DefaultRouter()
+router.register(r'search', StartupSearchViewSet, basename='startup-search')
```

Comment:
For consistency, consider using plural or more descriptive basenames, e.g., `startups-search`.